evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -416,9 +416,7 @@ contract FluxAggregator is AggregatorInterface, Owned {
    * that they're receiving fresh data by inspecting the updatedAt and
    * answeredInRound return values. Consumers are encouraged to
    * use this more fully featured method over the "legacy" getAnswer/
-   * latestAnswer/getTimestamp/latestTimestamp functions. Consumers are
```

Comment:
@hellobart I don't see where this is duplicated? Am I missing something?

Reply:
See: https://github.com/smartcontractkit/chainlink/blob/e9dea8a59460cf9ed8cd2162858e6f5682d2563b/evm-contracts/src/v0.6/dev/FluxAggregator.sol#L432